### DISCUSSION TITLE
Blitz install command could have a different name

### DISCUSSION THREAD

**Discussion by kvnwolf** [blitz-js/blitz#1738] [Ideas]
Hi all! So far I am really liking this framework, thanks all for the great efforts!

## The Problem

I find the `blitz install` command to be more closely related with _only installing stuff_, (that was my first impression) while a recipe can do more things, including but not restricted to: install dependencies, add configuration files, run codemods, add files, remove files, update files, etc.

## The Solution

If the `blitz install` recipe would be named different, something like `blitz recipe`. In that way, users would know that something will happen on _their code_, not only _their dependencies_.

Examples:
- `blitz recipe cook tailwind`
- `blitz recipe cook sentry`
- `blitz recipe cook material-ui`

## Extra info

By creating a new `blitz recipe` command, extra features like `blitz recipe create` can be added to easily scaffold new recipes.

**Comment by malkomalko** [comment]
💯 to the `blitz recipe` prefix, but I think I'd prefer something like `blitz recipe add` instead of `cook`

```
blitz recipe add {recipe-name}
blitz recipe create {recipe-name}
blitz recipe ls
blitz recipe search {search-term}
```

Something like the above.
